docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/logo/logo.twig

```diff
@@ -41,12 +42,16 @@
         <span class="ct-logo__stripe {{ type == 'inline' ? 'hide-xxs show-l' : '' }}"></span>
       {% endif %}
       {% for breakpoint, img in logo %}
-        {% include '@atoms/image/image.twig' with {
-          theme: theme,
-          url: img.url,
-          alt: img.alt,
-          modifier_class: 'ct-logo__image ct-logo__image--' ~ breakpoint ~ ' ' ~ (breakpoint == 'mobile' ? (type == 'inline' and logo_type == 'secondary' ? 'hide-xxs' : 'hide-l') : 'hide-xxs show-l'),
-        } only %}
+        {% if img.svg_logo %}
```

Comment:
https://caniuse.com/svg-img

Please revert this and pass SVG file directly to `<img>` tag